tests/test_nb_run.py

```diff
@@ -43,7 +44,7 @@ def test_failing(self, testdir2: Never):
         run = NotebookRun(filename, 300)
         res: NotebookResult = run.execute()
 
-        assert res.error and res.error.failing_cell_index == 1
```

Comment:
what was this issue here? I can see that it should be cell 0 failing, but why was this passing?